src/GitTools.IssueTrackers/AuthSettings.cs

```diff
@@ -0,0 +1,38 @@
+namespace GitTools.IssueTrackers
+{
+    using System;
+    using JetBrains.Annotations;
+
+    public class AuthSettings
+    {
+        public AuthSettings()
+        {
+            IsEmpty = true;
+        }
+
+        public AuthSettings([NotNull] string username, [NotNull] string password)
```

Comment:
Its 3 properties. And issue tracker auth is different to git auth. You don't authenticate to your issue tracker with SSH. We also drop the libgit2sharp dependency which is not needed in this package :)

Trying to overreuse is just as bad. GitTools.Core for me has some clear goals, which is dynamic repos, repo normalisation and other complex git related things.

If we find we need some common data structures then maybe we have GitTools.Common down the track.